### ISSUE TITLE
WCAG 2.1 SCs with incomplete/no techniques

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1813]
Many of the 2.1 SCs are still missing some/all techniques. Some have the `@@` placeholder indicator

[1.3.6 Identify Purpose (AAA)](https://www.w3.org/WAI/WCAG21/Understanding/identify-purpose.html)

Has the following sufficient techniques

> * Programmatically indicating the purpose of icons, regions and user interface components
> * [...]
> * Using microdata to markup user interface components (future link)

Under advisory techniques, these could do with some actual written-out technique (assuming there's an actual working model)

> * Enabling user agents to find the version of the content that best fits their needs
> * Using semantics to identify important features (e.g., coga-simplification="simplest")
> * [...]

---

[1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

Lots of `@@` in sufficient and advisory techniques.

---

[1.4.13 Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)

Lots of `@@`, and some of the proposed techniques there don't seem to actually solve the issue

> * @@ ARIA: Using role="tooltip"
> * @@ CSS: Using hover and focus pseudo classes

neither of those would address the core ask of the SC.

---

[G217: Providing a mechanism to allow users to remap or turn off character key shortcuts](https://www.w3.org/WAI/WCAG21/Techniques/general/G217) (for 2.1.4: Character Key Shortcuts) has at the end

> Resources are for information purposes only, no endorsement implied.
> 
> * Link

---

[2.3.3 Animation from Interactions](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)

> * Gx: Allowing users to set a preference that prevents animation.

---

[2.5.2 Pointer Cancellation](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html)

> * @@ Touch events are only triggered when touch is removed from a control.

---

[2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)

Lots of `@@` sufficient and advisory techniques

---

[2.5.4 Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)

Sufficient techniques

> * GXXX: Supporting system level features which allow the user to disable motion actuation

Advisory techniques

> * FXXX: Failure of Success Criterion 2.5.4 due to disrupting or disabling system level features which allow the user to disable motion actuation

---

[2.5.6 Concurrent Input Mechanisms](https://www.w3.org/WAI/WCAG21/Understanding/concurrent-input-mechanisms.html)

Sufficient techniques

> * @@ Only using high-level, input-agnostic event handlers (focus, blur, click) in Javascript (operating systems/UAs fire these events for all input mechanisms).
> * @@ Registering event handlers for keyboard/keyboard-like and pointer inputs simultaneously in Javascript (see Example 1 in Pointer Events Level 2)

---

[4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

5 `(future link)` techniques

**Comment by yatil** [comment]
People are confused by the @@ signs. Either use "future technique", which is at least descriptive, or remove them.
